build-tools/automation/yaml-templates/shut-down-emulator.yaml

```diff
@@ -0,0 +1,19 @@
+parameters:
+  releaseImage: false    # 'true' to release a specific emulator, if 'true', other 'release*' parameters must be provided
+  releaseDeviceName:     # Device name to release, like 'wear_square'
+  releaseAvdApiLevel:    # Device API level to release, like '30'
+  releaseAvdAbi:         # Device ABI to release, like 'x86'
+  releaseAvdType:        # Device AVD to release, like 'android-wear'
+
+steps:
+# This uses our in-tree 'dotnet' because the system one may not have the 'android' workload installed.
+- template: run-dotnet-preview.yaml
+  parameters:
+    displayName: Shut down emulator
+    condition: always()
+    continueOnError: true
+    project: tests/Mono.Android-Tests/Runtime-Microsoft.Android.Sdk/Mono.Android.NET-Tests.csproj
```

Comment:
We should be able to use https://github.com/xamarin/xamarin-android/blob/main/src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/Emulator.csproj instead of one of the Android application projects to avoid having to use a custom MSBuild. I didn't see this file when I initially authored this.

Reply:
Nice, that is much better!